### PR TITLE
[Wiki] Remove incorrect "optional" word

### PR DESCRIPTION
**Pull request by AntonMenov**
### Type

- [ ] Internal change (Doesn't affect end-user).
- [ ] External change (Does affect end-user).
- [x] Wiki (Changes towards the [Wiki]).
- [ ] Other: __________ 

### Description

This commit removes the incorrect word "optional" in _depend_ comment. Before this commit, both _softdepend_ and _depend_ had the same "optional" comment of.

Closes N/A 

[Wiki]: https://github.com/PlaceholderAPI/PlaceholderAPI/wiki

### CODE REVIEW

File: Hook-into-PlaceholderAPI.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ Next step is to go to your plugin.yml or paper-plugin.yml and add PlaceholderAPI
   author: author
   main: your.main.path.Here
   
-  # This sets PlaceholderAPI as an optional dependency for your plugin.
+  # This sets PlaceholderAPI as a dependency for your plugin.
```
Comment (Andre601) [inline Hook-into-PlaceholderAPI.md:97]: What about adding that it makes it required?
Would that be too much?

Comment (AntonMenov) [reply Hook-into-PlaceholderAPI.md:97]: It's hard to say which is better. In my opinion, "mandatory" sounds better than "required", but perhaps it's still too much.

Comment (Andre601) [reply Hook-into-PlaceholderAPI.md:97]: Yeah. The current change is fine enough. We can consider a change for the future perhaps...

**Review by Andre601 — APPROVED** [review]

**Comment by Andre601** [comment]
Thanks for the correction.

**Pull request merged**